modules/client-configuration/pages/supported-features.adoc

```diff
@@ -111,12 +111,22 @@ The operating system you run on a client is supported by the organization that s
 | {check}
 
 | {centos} 8
+ifeval::[{uyuni-content} == true]
+| {x86_64}, {ppc}
```

Comment:
I am *not* convinced at all about how {ppc} gets replaced. It gets replaced with `POWER` but what does that mean?

First of all, I guess we mean PowerPC and not Power, but even so: ppc32? ppc64? ppc64le?

In the end from the whole Power family, we only support ppc64le (PowerPC 64 bits little endian) for SLE (with the exception of SLE that I think it still supports ppc -32bit- and ppc64 -big endian-)

FMPOV this is specially important because at the call to `spacewalk-common-channels` you cannot use `POWER`.

@paususe @mcalmer any thoughts?

Reply:
Just for the record, I see the problem with the mention to `ARM` for SLE:

![image](https://user-images.githubusercontent.com/4226070/96746590-a013bd80-13c7-11eb-9085-de35a93892e7.png)

Reply:
We have {ppc} and {ppc64le} and both get a replacement that's only useful for "marketing" materials but not for command-line tools or technical information: "POWER" and "Power PC" respectively. 

We should check where each one of those is used and use either "POWER" (marketing, e. g. the table above), "ppc" (technical details and CLI tools for SLE 10), "ppc64" (technical details and CLI tools for SLE 11) or "ppc64le" (technical details and CLI tools SLE 12+).

Reply:
We have these:

```
    ppc: POWER
    ppc64le: 'IBM Power PC'
```

I agree with Pau, we need to expand the list. If you don't want to do it, just create an issue and I'll do it for you.

Reply:
I think pure PPC is only available for SLE11. Since SLE12 we only support ppc64le.

Reply:
SLE11 is ppc64
SLE10 is ppc

Reply:
Ah, so ppc from https://build.suse.de/project/show/SUSE:SLE-11-SP4:Update is not used? I see both there, ppc and ppc64

Reply:
> I agree with Pau, we need to expand the list. If you don't want to do it, just create an issue and I'll do it for you.

If I understand we now want to have:

>     ppc: 'IBM Power PC'
>     ppc64: ppc64
>     ppc64le: ppc64le

The first one to be used for "Marketing", the second one to be used for technical things such as the matrix.

Is that right?

Reply:
For marketing purposes:
* "IBM Power PC" (or maybe even shorter: "Power PC") should only be used for SLE 10 SP3, which we no longer support (but client tools are still available from SCC)
* "IBM POWER" for SLE11+, CentOS/RHEL/OL 6+

For technical details:
* SLE 10 SP3: ppc
* SLE 11 SP4, CentOS/RHEL/OL 6: ppc64
* SLE 12+, CentOS/RHEL/OL 7+: ppc64le

Reply:
> For marketing purposes:
> 
> - "IBM Power PC" (or maybe even shorter: "Power PC") should only be used for SLE 10 SP3, which we no longer support (but client tools are still available from SCC)

Yeah, but SLE10 is not documented anywhere anymore, so no need to worry about it.

> - "IBM POWER" for SLE11+, CentOS/RHEL/OL 6+

OK. Just remember we will NOT add ppc64 for CentOS/RHEL/OL6. EoL is almost here. Only 7 and 8 was requested.

But I see a problem with what you said before: you see the matrix as marketing, I see it as technical. I'd really want to use the technical names on the matrix, so the users an use that anywhere else, whey adding repos, syncing, whatever. If we do not do that, then that means that we need to *duplicate* the information of the architectures. Either that or we have footnotes on the matrix explaining what each "marketing name" means.

Which brings me to the next question: we mention on the matrix Itanium, not ia64, IBM Z, and not s390x... and so on. Shouldn't we do the same?

I think this really calls for a separate card, yes.

> For technical details:
> 
> - SLE 10 SP3: ppc

Not documented, so not really needed.

> - SLE 11 SP4, CentOS/RHEL/OL 6: ppc64

OK. Just as said above: no ppc64 on CentOS/RHEL/OL6

> - SLE 12+, CentOS/RHEL/OL 7+: ppc64le

OK.

Reply:
Let's get this PR merged, as the discussion is affecting other architectures as well.

I will create a card.

Reply:
https://github.com/SUSE/spacewalk/issues/12836